ansible/playbook/openstack_vm_create_aggregate.yml

```diff
@@ -0,0 +1,32 @@
+---
+# Variables:
+#  - "vm_name": "n311-test"
+#  - "openstack": {
+#      "vm": {
+#        "network": "provider_net_shared"
+#      }
+#    }}
+#  . k8s_type: Kubernetes host type [masters,nodes], empty for no k8s installation
+#  . k8s_version: Kubernetes version [117 ... 121], empty for no k8s installation
+
+- name: "Initialize passwordstore inventory"
+  import_playbook: "passstore_controller_inventory.yml"
+  vars:
+    vm_user: "centos"
+    pass_provider: "openstack"
+  tags: [create]
+
+- name: "Create Server on Openstack"
+  import_playbook: "openstack_vm_create.yml"
+  vars:
+    hetzner_context_name: "snowdrop"
```

Comment:
Why do we have a var `hetzner_context_name` part of this playbook ?

Reply:
removed this reference as it wasn't being used